configuration/src/main/kotlin/com/malinskiy/marathon/config/Configuration.kt

```diff
@@ -73,7 +75,8 @@ data class Configuration private constructor(
             "debug" to debug.toString(),
             "screenRecordingPolicy" to screenRecordingPolicy.toString(),
             "vendorConfiguration" to vendorConfiguration.toString(),
-            "deviceInitializationTimeoutMillis" to deviceInitializationTimeoutMillis.toString()
+            "deviceInitializationTimeoutMillis" to deviceInitializationTimeoutMillis.toString(),
+            "buildSystem" to buildSystem
```

Comment:
This probably should be a separate configuration class, e.g. BuildSystemConfiguration -> TeamCityConfiguration instead of just an enum. Otherwise we don't make it extensible in the future with parameters to the build system (e.g. ignore X, print more, etc)